packages/fuel-gauge/src/predicate/predicate-populate-witness.test.ts

```diff
@@ -52,52 +33,45 @@ describe('Predicate', () => {
         } as Required<ExcludeResourcesOption>
       );
 
-    let quantity: CoinQuantityLike[];
-    beforeAll(async () => {
-      provider = await Provider.create(FUEL_NETWORK_URL, { cacheUtxo: 1000 });
-      baseAssetId = provider.getBaseAssetId();
-      quantity = [[500, baseAssetId]];
-      wallet1 = Wallet.generate({ provider });
-      wallet2 = Wallet.generate({ provider });
-      wallet3 = Wallet.generate({ provider });
-      receiver = Wallet.generate({ provider });
-      predicate1 = new Predicate<[number]>({
-        bytecode: assertNumberArtifacts.binHexlified,
-        provider,
-        abi: assertNumberArtifacts.abiContents,
-        inputData: [11],
+    it('should properly populate predicate data and remove placeholder witness [CASE 1]', async () => {
+      using launched = await launchTestNode({
+        walletsConfig: {
+          amountPerCoin: 100_000_000,
+        },
       });
+      const {
+        provider,
+        wallets: [wallet],
+      } = launched;
+
+      const quantity: CoinQuantityLike[] = [[500, provider.getBaseAssetId()]];
 
-      predicate2 = new Predicate<[boolean]>({
-        bytecode: assertValueArtifacts.binHexlified,
-        abi: assertValueArtifacts.abiContents,
+      const predicateAssertNumber = new Predicate<[number]>({
+        bytecode: PredicateAssertNumberAbi__factory.bin,
+        abi: PredicateAssertNumberAbi__factory.abi,
         provider,
-        inputData: [true],
+        inputData: [11],
       });
 
-      await seedTestWallet(
-        [wallet1, wallet2, wallet3, predicate1, predicate2],
-        [[500_000_000, baseAssetId]],
-        UTXOS_AMOUNT
-      );
```

Comment:
Might be worth using the `seedTestWallet` functionality over the `launchTestNode` wallet config for now. Given that we're unable to separate by assets into separate UTXOS.